StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/NetworkingOTPView/NetworkingOTPView.swift

```diff
@@ -46,7 +46,16 @@ final class NetworkingOTPView: UIView {
     // TODO(kgaidis): make changes to `OneTimeCodeTextField` to
     // make the font larger
```

Comment:
nit, does this resolve the TODO so it can be rm'd?

Reply:
thank you great catch; removed!